app/server.rb

```diff
@@ -0,0 +1,79 @@
+require 'sinatra/base'
+require 'data_mapper'
+require_relative 'helper'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/chitter_#{env}")
+
+require_relative 'models/message'
+require_relative 'models/user'
+
+DataMapper.finalize
+
+DataMapper.auto_upgrade!
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set :session_secret, 'secret'
+
+  get '/' do
+    @messages = Message.all
+    erb :index
+  end
+
+  post '/' do 
+    erb :index
+  end
+
+  get '/users/new' do 
+    @user = User.new
+    erb :"/users/new"
+  end
+
+  post '/users/new' do 
+    @user = User.create(:name => params[:name], :email => params[:email], :password => params[:password])
```

Comment:
Metrics/LineLength: Line is too long. [105/90]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.